tests/e2e/instance_test.go

```diff
@@ -0,0 +1,59 @@
+package e2e_test
+
+import (
+	"testing"
+
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	appConsumer "github.com/cosmos/interchain-security/app/consumer"
+	appConsumerDemocracy "github.com/cosmos/interchain-security/app/consumer-democracy"
+	appProvider "github.com/cosmos/interchain-security/app/provider"
+	"github.com/cosmos/interchain-security/tests/e2e"
+	e2etestutil "github.com/cosmos/interchain-security/testutil/e2e"
+	"github.com/cosmos/interchain-security/testutil/simapp"
+	"github.com/stretchr/testify/suite"
+)
+
+// This file can be used as an example e2e testing instance for any provider/consumer applications.
```

Comment:
This is a really nice overview on how to use this and it communicates the requrements in a clear and concise manner.

Maybe consider adding it to the `tests/e2e` readme.

Reply:
Agreed, I've added instructions to the top level readme, and the `tests/e2e` has a quick mention as well